notebook/mlops.ipynb

```diff
@@ -4,28 +4,57 @@
    "cell_type": "markdown",
```

Comment:
<p>Perhaps this is where we can talk about if its the first time there will be no deployment application, but if this is the second time, there will be a canary deployment which you can click through to - I think it could be useful to include some screen shots here on how to view the Canary Deployment given there are a couple of clicks to go through?</p><p> </p><p>Also we could include the image of the traffic shifting here once the deployment is running (https://github.com/aws-samples/amazon-sagemaker-safe-deployment-pipeline/blob/master/docs/code-deploy.gif)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws-samples/amazon-sagemaker-safe-deployment-pipeline/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>

Reply:
<p>Thanks, I'll change the section based on these suggestions. I'm also wondering if it would make sense to just move this whole section down below the "trigger retraining" section? There doesn't seem to be much value in any part of this section until there are two models. </p>

Reply:
<p>Yes - although i'm proposing we now link to the lambda deployment - which will be easier to jump through to code deploy - and it still will show a version for the first deployment which will still make sense.</p><p> </p><pre class="ql-syntax" spellcheck="false">HTML('&lt;a target="_blank" href="https://{0}.console.aws.amazon.com/lambda/home?region={0}#/applications/{1}-deploy-prd?tab=deploy"&gt;Lambda Deployment&lt;/a&gt;'.format(region, model_name))
</pre><p> </p>